### ISSUE TITLE
librarian-chef install - undefined method `correct?' for nil:NilClass

### ISSUE THREAD

**Issue by mwhooker** [applicationsonline/librarian-chef#12]
stack trace

```
/Users/mwhooker/.rbenv/versions/1.9.3-p392/lib/ruby/gems/1.9.1/gems/librarian-0.1.0/lib/librarian/action/persist_resolution_mixin.rb:11:in `persist_resolution': undefined method `correct?' for nil:NilClass (NoMethodError)
    from /Users/mwhooker/.rbenv/versions/1.9.3-p392/lib/ruby/gems/1.9.1/gems/librarian-0.1.0/lib/librarian/action/resolve.rb:27:in `run'
    from /Users/mwhooker/.rbenv/versions/1.9.3-p392/lib/ruby/gems/1.9.1/gems/librarian-0.1.0/lib/librarian/cli.rb:169:in `resolve!'
    from /Users/mwhooker/.rbenv/versions/1.9.3-p392/lib/ruby/gems/1.9.1/gems/librarian-chef-0.0.1/lib/librarian/chef/cli.rb:41:in `install'
    from /Users/mwhooker/.rbenv/versions/1.9.3-p392/lib/ruby/gems/1.9.1/gems/thor-0.18.1/lib/thor/command.rb:27:in `run'
    from /Users/mwhooker/.rbenv/versions/1.9.3-p392/lib/ruby/gems/1.9.1/gems/thor-0.18.1/lib/thor/invocation.rb:120:in `invoke_command'
    from /Users/mwhooker/.rbenv/versions/1.9.3-p392/lib/ruby/gems/1.9.1/gems/thor-0.18.1/lib/thor.rb:363:in `dispatch'
    from /Users/mwhooker/.rbenv/versions/1.9.3-p392/lib/ruby/gems/1.9.1/gems/thor-0.18.1/lib/thor/base.rb:439:in `start'
    from /Users/mwhooker/.rbenv/versions/1.9.3-p392/lib/ruby/gems/1.9.1/gems/librarian-0.1.0/lib/librarian/cli.rb:26:in `block (2 levels) in bin!'
    from /Users/mwhooker/.rbenv/versions/1.9.3-p392/lib/ruby/gems/1.9.1/gems/librarian-0.1.0/lib/librarian/cli.rb:31:in `returning_status'
    from /Users/mwhooker/.rbenv/versions/1.9.3-p392/lib/ruby/gems/1.9.1/gems/librarian-0.1.0/lib/librarian/cli.rb:26:in `block in bin!'
    from /Users/mwhooker/.rbenv/versions/1.9.3-p392/lib/ruby/gems/1.9.1/gems/librarian-0.1.0/lib/librarian/cli.rb:47:in `with_environment'
    from /Users/mwhooker/.rbenv/versions/1.9.3-p392/lib/ruby/gems/1.9.1/gems/librarian-0.1.0/lib/librarian/cli.rb:26:in `bin!'
    from /Users/mwhooker/.rbenv/versions/1.9.3-p392/lib/ruby/gems/1.9.1/gems/librarian-chef-0.0.1/bin/librarian-chef:7:in `<top (required)>'
    from /Users/mwhooker/.rbenv/versions/1.9.3-p392/bin/librarian-chef:23:in `load'
    from /Users/mwhooker/.rbenv/versions/1.9.3-p392/bin/librarian-chef:23:in `<main>'
```

**Comment by gregkare** [comment]
This is caused by a bug in librarian 0.1.0. There's a fix and it was merged but there hasn't been a release yet: https://github.com/applicationsonline/librarian/pull/145

**Comment by yfeldblum** [comment]
This is released in `librarian-v0.1.1`.
